erpnext/stock/doctype/delivery_note/delivery_note.py

```diff
@@ -301,6 +301,24 @@ def update_billing_status(self, update_modified=True):
 
 		self.load_from_db()
 
+	def get_batched_serialized_details(self, item_code):
```

Comment:
So Stock Entry simply had a comprehensive `get_item_details` method that got many things. Delivery Note and the Invoices don't have such method. This is the way I fetched whether the item is batched or serialised here. But doing the same thing for the invoices seems too repetitive and specialised.